core/src/main/java/io/questdb/cairo/TableReader.java

```diff
@@ -824,15 +810,16 @@ private ReadOnlyColumn openOrCreateMemory(Path path, ObjList<ReadOnlyColumn> col
     }
 
     private long openPartition0(int partitionIndex) {
-        // is this table is partitioned?
-        if (timestampAddMethod != null
-                && removedPartitions.contains(timestampAddMethod.calculate(
-                minTimestamp, partitionIndex
-        ))) {
-            return -1;
+        // is this table partitioned?
+        long partitionTimestamp = 0;
+        if (timestampAddMethod != null) {
```

Comment:
this method should not be translating index to timestamp and then converting timestamp back to index